docs/dev/clusteroperator.md

```diff
@@ -0,0 +1,63 @@
+# ClusterOperator Custom Resource
+
+The ClusterOperator is a custom resource object which holds the current state of an operator. This object is used by operators to convey their state to the rest of the cluster.
+
+Ref: [godoc](https://godoc.org/github.com/openshift/cluster-version-operator/pkg/apis/operatorstatus.openshift.io/v1#ClusterOperator) for more info on the ClusterOperator type.
+
+## Why I want ClusterOperator Custom Resource in /manifests
+
+ClusterVersionOperator sweeps the release payload and apply it to the cluster.
```

Comment:
nit: "apply" -> "applies".  Also, this sentence leads into the next; they should probably go in the same paragraph.